src/ifop/clients.ts

```diff
@@ -2,10 +2,20 @@ import {NextFunction, Request, Response} from "express";
 import * as ifas from "../ifas/auth";
 import { Logger } from "../utils/logger";
 
+import userDB from "../config/users.json";
+
 const deviceDB = {
 
 };
 
+export function initDeviceDB() {
+  for (const userName in userDB) {
+    if (userDB.hasOwnProperty(userName)) {
+      deviceDB[getKey(userName, userDB[userName].deviceID)] = true;
+    }
+  }
+}
+
```

Comment:
why is this needed? i would like to hold user info only in ifas

Reply:
That was just for the test setup, I'm gonna remove it.